src/LSDStakingNode.sol

```diff
@@ -17,6 +17,8 @@ interface ILSDStakingNodeEvents {
     event DepositToEigenlayer(IERC20 indexed asset, IStrategy indexed strategy, uint256 amount, uint256 eigenShares);
```

Comment:
can skip this file for now we're probably not going to include this in this release